src/workloads/execution/TimeSeriesArbitraryUpdate.yml

```diff
@@ -55,9 +55,10 @@ AutoRun:
 - When:
     mongodb_setup:
       $eq:
-      - shard-lite
-      - replica
+      # TODO (PERF-4726): Enable on these variants.
+      # - shard-lite
+      # - replica
```

Comment:
Can we leave these uncommented since we bumped the version to 7.2?

Reply:
Discussed offline and will turn off the feature flag on 7.2 as well. So this will stay as is.